<launch>
    <include file="$(find flo_face)/launch/base.launch" />
    <node name="bolide_interface" pkg="flo_humanoid" type="robot_controller.py" />

    <arg name="model" default="$(find flo_humanoid)/urdf/flo.urdf.xacro"/>
    <param name="robot_description" command="$(find xacro)/xacro.py --inorder $(arg model) " />
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />

</launch>
